cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -552,6 +471,18 @@ public static KafkaCluster fromCrd(Reconciliation reconciliation, Kafka kafkaAss
         return result;
     }
 
+    public static List<String> generatePodList(String cluster, int replicas) {
+
+        ArrayList<String> podNames = new ArrayList<>(replicas);
+
+        for (int podId = 0; podId < replicas; podId++) {
+
+            podNames.add(KafkaResources.kafkaPodName(cluster, podId));
+
+        }
+        return podNames;
```

Comment:
too many blank lines?

Reply:
If it is fine with you, I will fix this in the next PR.

Reply:
yeah ok it was a nit anyway

Reply:
Thanks, I appreciate it. I already started working on the KAO refactoring on top of this, so it would be complicated for me to change this here and then backport it there.